ArduPlane/GCS_Mavlink.cpp

```diff
@@ -699,6 +699,8 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
     case MSG_AOA_SSA:
         CHECK_PAYLOAD_SIZE(AOA_SSA);
         plane.send_aoa_ssa(chan);
+    case MSG_LANDING:
+        plane.landing.send_landing_message(chan);
```

Comment:
How could you? You don't know what message a different backend will need... You could assume the deepstall one for now, but thats not really robust....